pennylane/devices/default_tensor.py

```diff
@@ -258,6 +266,9 @@ def circuit(num_qubits):
 
     We can provide additional keyword arguments to the device to customize the simulation. These are passed to the ``quimb`` backend.
 
+    .. warning::
```

Comment:
```suggestion
    .. note::
        Be aware that `quimb` uses multi-threading with `numba <https://numba.pydata.org/numba-doc/dev/user/threading-layer.html>`_ as well as for linear algebra operations with `numpy.linalg <https://numpy.org/doc/stable/reference/routines.linalg.html#linear-algebra-numpy-linalg>`_. Proper setting of the corresponding environment variables (e.g. `OMP_NUM_THREADS`, `OPENBLAS_NUM_THREADS`, `NUMBA_NUM_THREADS` etc.) depending on your hardware is highly recommended and will have a strong impact on the device's performance.
```
I don't know what to tell about https://quimb.readthedocs.io/en/main/autoapi/quimb/core/index.html#quimb.core._NUM_THREAD_WORKERS and if worth mentioning.

Reply:
The `NUM_THREAD_WORKERS` is defeined by `OMP_NUM_THREADS` ([ref](https://github.com/jcmgray/quimb/blob/ed21567aac692ee68abde76b8d124ff9d701b081/quimb/core.py#L28-L39)) then is not worth mentioning.